virttest/qemu_vm.py

```diff
@@ -3022,6 +3023,10 @@ def _cleanup(self, free_mac_addresses):
             except OSError:
                 pass
 
+        nic = self.virtnet[0]
```

Comment:
we can't add it here,  if VM use a static IP address, it will remove IP/MAC mapping,  so add it in wait_for_get_address  looks more reasonable.